### PR TITLE
SCW Verification

### PR DESCRIPTION
**Pull request by codabrink**

### CODE REVIEW

File: proto/mls/message_contents/association.proto [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// Associations and signatures
```
Comment (neekolas) [inline proto/mls/message_contents/association.proto:1]: I think we might be able to leave this file deleted

File: proto/mls/message_contents/credential.proto [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Credentials and revocations
```
Comment (neekolas) [inline proto/mls/message_contents/credential.proto:1]: I _believe_ we can leave this deleted too

File: proto/mls/message_contents/association.proto (hunk 1 above)
Comment (codabrink) [reply proto/mls/message_contents/association.proto]: Cool

File: proto/mls/message_contents/credential.proto (hunk 2 above)
Comment (codabrink) [reply proto/mls/message_contents/credential.proto]: Gone

**Review by insipx — APPROVED** [review]

**Pull request merged**